sdks/python/apache_beam/examples/inference/sklearn_inference_example.py

```diff
@@ -0,0 +1,98 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import argparse
+import numpy as np
+
+import apache_beam as beam
+
+from apache_beam.ml.inference.api import RunInference
+from apache_beam.ml.inference.sklearn_inference import ModelFileType
+from apache_beam.ml.inference.sklearn_inference import SklearnModelLoader
+from apache_beam.options.pipeline_options import PipelineOptions
+from apache_beam.options.pipeline_options import SetupOptions
+
+
+class GetData(beam.DoFn):
+  def process(self, dataset_name):
+    if dataset_name == 'fetch_20newsgroups':
+      from sklearn.datasets import fetch_20newsgroups_vectorized
+      newsgroups_test = fetch_20newsgroups_vectorized(
+          subset='test', data_home='/tmp')
+      for x, y in zip(newsgroups_test['data'], newsgroups_test['target']):
+        # (n_feat,) is supported but not (1, n_feat)
+        yield y, np.squeeze(x.toarray())
+    else:
+      raise NotImplementedError
+
+
+class PostProcessor(beam.DoFn):
+  """Post process PredictionResult to output true_label and
```

Comment:
Maybe rename true_label to expected_label or just label. ML people should know what that is.

Reply:
conventional way, we call label as true_label. Anyway, I am gonna change the docstring on this method soon